api-reference/v2/live/message/post-transcript.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: Post Transcript
```

Comment:
```suggestion
title: Transcript
```
To be consistent with others that removed the prefix "Post"

Reply:
I think this one refers to post transcript event, not `POST transcript`